### PR TITLE
First Patreon Characters Crusade

### PR DESCRIPTION
**Pull request by Gaben1025**
ℹ️ **Development Notice**: Please note that this branch is based on [1-8-1-update](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/tree/1-8-1-update), which means it **_should not_** be merged into `master` until after the `1-8-1-update` branch has been merged.

### Links:
- 📗 [Life-Binder Codex of Eonar](https://docs.google.com/spreadsheets/d/1hKQmfMrI_d8FtRez5w68OfS1s0JVr5r2yF8nryGe104)

## Changelog:
- Added **~46** Patreon characters.

### Draenei

**Courtiers**
- Haera Forcythe (558988b2f39cf45438b69fa9d050d1189a3e762d)

### Dwarves

**Counts**
- Galgamn Icemane (4f2b6e4843e6115fc58bebbaf03db7ac6e54c745)

**Barons**
- Vikram Weihbart (86ccf1a90c06dba0832dcad35843fa3ea7c48d5b)

**Courtiers**
- Dunbarten Varlas (7df1928dd97bf95e8eceb5fee12f28834d5745fb)
  - Traits may need to be balanced. Customization points (573) exceed achievement limit by 173 points.

### Gnolls

**Courtiers**
- Grün Timberpaw (558988b2f39cf45438b69fa9d050d1189a3e762d)
  - Requested dynasty name Woodpaw changed to Timberpaw because Woodpaw already exists.

### Goblins

**Counts**
- Ryzzik Mushdough (42d0f671fef57ba5314ad3c125e2f407f1fd9f6c)

### Highborne

**Courtiers**
- Nuthurdel Aastran (836dfdf1c80d0e7275ff5e60615d25b9e5a536ba)
  - Traits may need to be balanced. Customization points (603) exceed achievement limit by 203 points.

### High Elves

**Counts**
- Mahiro Shadyneth (67b9bab68f5a52008127c3de6140b8d3b896b3b8)
- Kalandros Ironbane (f1d49b3df7a42db5e30e78b4baca2c226f6c4bc5)
- Vallesta Shan (da28ebfc59790365a6e66d13b3c99dae293baf9b)
- Ralanor Arnoldian (0680dd4626de6161167610e272856eec27f52909)
- Salazard Blacksun (81d2bd53851d9b2f794196d5c9264e73f51877ca)

**Courtiers**
- Aeraeniel Silvermist (81d2bd53851d9b2f794196d5c9264e73f51877ca)
- Anarial Dawnstrike (d79a23796eec68783ae269d014e7e412ab54b4c0)
  - Traits may need to be balanced. Customization points (563) exceed achievement limit by 163 points.
- Shar Runeguard (998e1d8d4e90d79b3038603f7acb64c1bd101ac4)
  - Traits may need to be balanced. Customization points (428) exceed achievement limit by 28 points.
- Harku Sundream (cdfb34af15ea547167cf603d0c9cc488f1c21339)
  - Traits may need to be balanced. Customization points (493) exceed achievement limit by 93 points.

**Unknown**
- Kireina Radiant Blade (a15a11a1e17e49f5c9d5cb5f9b78665c61341e6a)

### Humans

**Counts**
- Eddard Locke (558988b2f39cf45438b69fa9d050d1189a3e762d)
- Fyodor Morarenko (4515c9699b0fa52e252ca2d8e62ab0f846649f2b)
- Nicole Broomford (ebb682c2de95d766383313287d790b0fa0a1eae1)

**Courtiers**
- Trystan Redcap (ffd420a6271409b03182f4a03c14c02e3abef755)
  - Traits may need to be balanced. Customization points (613) exceed achievement limit by 213 points.
- Spasmus Brainwave (ac8d602099a174f129d229cf45210e1541084fd6)
- Thorias Eldaren (30cb0fb3638596248dd52af61d8afca3b347e921)
- Carmelia Kerlf (d1924095377b663404acffd1b57cd0f7839ad60c)
  - Traits may need to be balanced. Customization points (573) exceed achievement limit by 173 points.
- Sallion Seraphis (a85f5efc3051dc09f3bcccf4f30577118db84cf7)
- Lilith Nightshade (18210a7b7c6680990826769262adaad6b39bf81a)
  - Traits may need to be balanced. Customization points (503) exceed achievement limit by 103 points.
- Nathaniel Riverhine (836dfdf1c80d0e7275ff5e60615d25b9e5a536ba)
- Sahala Silver (564f0140e5be2c41293c13d48b60537cd75a1465)
  - Adjusted age by 1 year as she was too young to receive her education trait in the requested bookmark.
- Fyn LeFleur (60a3702a47ddef0c47290d516c42a8c5943938d0)
- Attha Lionheart (72b62b42176dbee56aaf375593d816c80e36002c)
  - Traits may need to be balanced. Customization points (428) exceed achievement limit by 28 points.
- Richard Aigle (9acd6bbf9ec84a227482a480af70dd4ace679702)
  - Traits may need to be balanced. Customization points (463) exceed achievement limit by 63 points.
- Zediwan Leodegar (427676f46b7834815f05cf00cdeb78ca7a8fa3a1)
  - Traits may need to be balanced. Customization points (568) exceed achievement limit by 168 points.

### Liches

**Counts**
- Kyr Darkcore (daf50746e6d6c873cdfc5d34f5ec88194640b2a3)

**Courtiers**
- Mughal Forsaken (2cbb0568d764290e179450710fa6efbadb796bb1)

### Murlocs

**Courtiers**
- Lrmlokr Murlochiavelli (80e6bb60518c38246ff89f0e81820caf2f5860e7)
  - Traits may need to be balanced. Customization points (473) exceed achievement limit by 73 points.

### Night Elves

**Counts**
- Thenell Shadowmoon (a16fdeec944578b1d974df70bb57eb0d3719a93a)
- Yeral An'arath (558988b2f39cf45438b69fa9d050d1189a3e762d)

**Courtiers**
- Anara Snowleaf (30cb0fb3638596248dd52af61d8afca3b347e921)
- Savaena Windsong (0efe6828d921d0e4dd172e52c285f05c96c7c25b)
- Grace Grace (1a2ae1987c945ef5deae66fdf7e10dffe4771c68)

### Orcs

**Courtiers**
- Kargatha Shadowglyph (814e27ab1c5af67d520dcff84f7376a3439fd721)
  - Traits may need to be balanced. Customization points (413) exceed achievement limit by 13 points.
- Thruk Corpserot (558988b2f39cf45438b69fa9d050d1189a3e762d)
- Durnack Hellfire (9acd6bbf9ec84a227482a480af70dd4ace679702)
  - Traits may need to be balanced. Customization points (543) exceed achievement limit by 143 points.

### Tauren

**Courtiers**
- Imhott Longhorn (619efe90a708cd5cd3b1bd5b4fd033c1f134d6f7)
- Chogan Steelbender (72b62b42176dbee56aaf375593d816c80e36002c)

### Vrykul

**Counts**
- Stolbond Hellebron (df9052edfaad4e8953574b25a02bee9c8004eb89)

## To-do list:
- [x] Give county to **Kalandros Ironbane** (ID: `patron310615`).
- [x] Give county to **Thenell Shadowmoon** (ID: `patron310616`).
- [x] Turn **Haera Forcythe** (ID: `patron310627`) into a Wanderer.
- [x] Balance the traits of **Anarial Dawnstrike** (ID: `patron310635`). Customization points (563) exceed achievement limit by 163 points.
- [x] Balance the traits of **Shar Runeguard** (ID: `patron310642`). Customization points (428) exceed achievement limit by 28 points.
- [x] Balance the traits of **Harku Sundream** (ID: `patron310643`). Customization points (493) exceed achievement limit by 93 points.
- [x] Balance the traits of **Nuthurdel Aastran** (ID: `patron310636`). Customization points (603) exceed achievement limit by 203 points.
- [x] Balance the traits of **Carmelia Kerlf** (ID: `patron310639`). Customization points (573) exceed achievement limit by 173 points.
- [x] Balance the traits of **Lilith Nightshade** (ID: `patron310644`). Customization points (503) exceed achievement limit by 103 points.
- [x] Balance the traits of **Lrmlokr Murlochiavelli** (ID: `patron310640`). Customization points (473) exceed achievement limit by 73 points.
- [x] Balance the traits of **Chogan Steelbender** (ID: `patron310649`). Customization points (433) exceed achievement limit by 33 points.
- [x] Balance the traits of **Attha Lionheart** (ID: `patron310650`). Customization points (428) exceed achievement limit by 28 points.
- [x] Balance the traits of **Richard Aigle** (ID: `patron310651`). Customization points (463) exceed achievement limit by 63 points.
- [x] Balance the traits of **Durnack Hellfire** (ID: `patron310652`). Customization points (543) exceed achievement limit by 143 points.
- [x] Balance the traits of **Zediwan Leodegar** (ID: `patron310653`). Customization points (568) exceed achievement limit by 168 points.
- [x] Balance the traits of **Dunbarten Varlas** (ID: `patron310634`). Customization points (573) exceed achievement limit by 173 points.
- [x] Balance the traits of **Trystan Redcap** (ID: `patron310613`). Customization points (613) exceed achievement limit by 213 points.
- [x] Balance the traits of **Kargatha Shadowglyph** (ID: `patron310611`). Customization points (413) exceed achievement limit by 13 points.
- [x] Check other characters for balance issues and note this in [the spreadsheet](https://docs.google.com/spreadsheets/d/1hKQmfMrI_d8FtRez5w68OfS1s0JVr5r2yF8nryGe104).

### CODE REVIEW

**Review by TheDrPibb1999 — CHANGES_REQUESTED** [review]
Ralanor Arnoldian is listed as a count on the document but is a courtier in game for all 3 starts. 
Capture

File: localization/german/dynasties/wc_dynasty_names_l_german.yml [hunk 1]
```diff
@@ -2828,6 +2827,52 @@
...
+ dynn_Snowleaf:0 "Snowleaf"
+ dynn_Timberpaw:0 "Timberpaw"
+ dynn_Corpserot:0 "Corpserot"
+ dynn_Forcythe:0 "Forcythe"
+ dynn_Anarath:0 "An'arath"
+ dynn_Locke:0 "Locke"
+ dynn_Weihbart:0 "Weihbart"
+ dynn_Blacksun:0 "Blacksun"
+ dynn_Silvermist:0 "Silvermist"
+ dynn_Morarenko:0 "Morarenko"
```
Comment (arithon) [inline localization/german/dynasties/wc_dynasty_names_l_german.yml:2854]: why is morarenko in a different place in the german file than in english + french?

File: localization/english/dynasties/wc_dynasty_names_l_english.yml [hunk 2]
```diff
@@ -2812,7 +2812,7 @@
  dynn_Forestcaller:0 "Forestcaller"
  dynn_Glarr:0 "Glarr"
  dynn_Winter:0 "Winter"
- dynn_Mace'arosh:0 "Mace'arosh"
+ dynn_Mace_arosh:0 "Mace'arosh"
```
Comment (arithon) [inline localization/english/dynasties/wc_dynasty_names_l_english.yml:2815]: needs to be changed in german, too

**Review by arithon — CHANGES_REQUESTED** [review]
some localisation changes required.

**Comment by hamjustine** [comment]
@TheGroxEmpire  Added Kireina Radiant Blade back and fixed missing dna for Aeraniel Silvermist

**Review by arithon — APPROVED** [review]
LGTM

**Review by TheGroxEmpire — APPROVED** [review]

**Pull request merged**